assets/css/_notification_drawer.scss

```diff
@@ -13,6 +13,13 @@
   @media screen and (max-width: $mobile-max-width) {
     width: 100%;
   }
+  @media screen and (max-width: $mobile-max-width) {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    width: 100%;
```

Comment:
It should have the same outcome, but I wonder if `100vw` would be a bit clearer about the fact that the intent is to take up all of the horizontal space on the screen. I don't think we're actually consistent one way or another in the existing CSS about when we use `vw` / `vh`.